requirements_test.txt

```diff
@@ -15,7 +15,9 @@ pydocstyle==4.0.1
 pylint==2.3.1
 astroid==2.2.5
 pytest-aiohttp==0.3.0
+pytest-asyncio==0.10.0
 pytest-cov==2.7.1
+pytest-localserver==0.5.0
```

Comment:
Is the local server actually doing I/O? We don't want to do I/O in tests. We want to patch and mock the library interfaces to avoid I/O but still be able to test things.

Reply:
It opens a local port (59800).
But there is no I/O to a remote server.

Reply:
Ok but it's still local I/O which we want to avoid.

I suggest we try to patch the library and use spec or autospec to get the correct interfaces.

https://docs.python.org/3/library/unittest.mock.html#auto-speccing

https://github.com/home-assistant/home-assistant/blob/4f55235aa2f6801b3b1e2a08ef2971df7bcb0c93/tests/components/hue/test_light.py#L178-L210

Reply:
What a pity. I close this PR and work on another solution.
Thanks for fast feedback.

Reply:
https://github.com/MartinHjelmare/home-assistant/blob/3cda3fb25857596e38de4c7a360c8903b00084c3/tests/components/mysensors/conftest.py#L63-L79